tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -683,6 +679,8 @@ def transform_param_to_str(value):
     'remove_memory': ParamDescription(
         "Removes the Memory layer and use additional inputs outputs instead", '', '',
         None),
+    'convert_to_fp16': ParamDescription(
+        'If original model is in FP32 all model weights and biases are compressed to FP16.', '', '', None),
```

Comment:
Actually, not all weights are converted to FP16. We have some exceptional cases to stay them in FP32, for example, eps constants. 
Let us thing about more accurate description

Reply:
@mvafin, @slyalin, will we start the deprecation process here in this PR? Let us mark `data_type` option as deprecated in this release and remove in the next.

Reply:
How about such description?
"If original model is in FP32 model weights and biases are compressed to FP16, except for const values which do not fit in FP16 range. All intermediate data is kept in original precision."
@rkazants @slyalin @mvafin

Reply:
As i mentioned below mixed FP16/FP32 are not going to this release. Let's remove mentioning that some values can be left in FP32 if `convert_to_fp16` is specified